src/main/api/2/studio-api-2.yaml

```diff
@@ -6330,6 +6330,197 @@ paths:
+                  type: array
+                  description: optional (soft) dependencies
+                  items:
+                    type: string
+                mandatory:
+                  type: array
+                  description: mandatory (hard) dependencies
+                  items:
+                    type: string
+                environment:
+                  type: string
+                  description: environment to publish content to
+                schedule:
+                  type: string
+                  format: datetime
+                  description: schedule when to publish content
+                submissionComment:
+                  type: string
+                  description: comment about publish request
+                sendEmail:
+                  type: boolean
+                  description: if true send email notifications
+              required:
+                - items
+                - environment
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  response:
+                    $ref: '#/components/schemas/ApiResponse'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+
+  /workflow/publish:
+    post:
+      tags:
+        - workflow
+      summary: Publish content
+      description: 'Required permission "publish"'
+      operationId: workflowPublish
+      requestBody:
+        description: parameters for publish content
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties: 
+                items:
+                  type: array
+                  description: path(s) of content item(s)
+                  items:
+                    type: string
+                optional:
+                  type: array
+                  description: optional (soft) dependencies
+                  items:
+                    type: string
+                mandatory:
+                  type: array
+                  description: mandatory (hard) dependencies
+                  items:
+                    type: string
+                environment:
+                  type: string
+                  description: environment to publish content to
+                schedule:
+                  type: string
+                  format: datetime
+                  description: schedule when to publish content
+                submissionComment:
+                  type: string
+                  description: submission comment by author
+                publishComment:
```

Comment:
On the UI, we only have one field in the dialog to enter a comment **and** the backend doesn't give us the previous submission comment to resend it on approval — nor do I see much point to it. Perhaps I'm misinterpreting, don't see why two different comments.

It helps to keep the two APIs congruent, so keeping same name for the comment on both APIs good. Perhaps just `comment`?